app/assets/stylesheets/nav_bar.scss

```diff
@@ -0,0 +1,145 @@
+.nav-container{
+    display: flex;
+    justify-content: center;
+    background-color: var(--primary-color);
+}
+nav{
```

Comment:
don't do a global style selector like this. Replace it with a class 

if you do this, it will affect all the `nav` in the application and we are not sure if it is only used in the top navbar.